api_functions/user.functions.js

```diff
@@ -15,4 +15,16 @@ const createUser = async (UserInfo) => {
   return user;
 };
 
-module.exports = { createUser };
+const getAllUsers = () => {
+  return User.find();
+};
+
+const getUser = (username) => {
+  return User.findOne({ name: username });
+};
+
+const deleteUser = (username) => {

```

Comment:
do we need to be able to delete? if not then we can just delete this imo

Reply:
I think the way I was thinking of it was that we'd add people to the DB each year, then when elections is over and its time to reset for next election cycle we clear our DB